src/main/resources/db/changelog/changesets/changelog_2022-06-01T12:46:54Z.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="boutiercha (generated)" id="1654087630098-9">
+        <createTable tableName="load_flow_parameters_entity_countries_to_balance">
+            <column name="load_flow_parameters_entity_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="countries_to_balance" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="boutiercha" id="1654087630098-10">
+        <addColumn tableName="load_flow_parameters">
+            <column name="connected_component_mode" type="varchar(255)"/>
+        </addColumn>
+        <sql splitStatements="true" stripComments="true">
+            UPDATE load_flow_parameters SET connected_component_mode = 'MAIN'
```

Comment:
why not using simply defaultValue attribute on <column> element just above ?

Reply:
Ok, I added the default value and removed the SQL statement.